.tools/bin/check-i18n

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env php
+<?php
+
+/**
+ * @author Thomas Blum
+ *
+ * This command checks for removable keys in i18n .lang files.
+ *
+ * Examples:
+ *
+ * bin/check-i18n
+ * bin/check-i18n core
+ * bin/check-i18n -addonName-
```

Comment:
Wo ich das hier so sehe.. eigentlich ist es doch auch für Addonentwickler nützlich, und könnte daher auch ein normaler Console-Command sein, den wir mit im Release ausliefern, oder?

Reply:
jetzt wo wir drüber reden, sehe ich es auch so.

Reply:
Wobei das nicht zwingend hier im PR sein muss. Können es auch erstmal so als Tool für dieses Repo aufnehmen, und irgendwann anders in einen richtigen Command umwandeln.
Also wie du Lust hast @tbaddade.

Reply:
Mache ich gern mit